### ISSUE TITLE
having issues with install.bat.

### ISSUE THREAD

**Issue by Kotonoha003** [noccu/umamusu-translate#56]
hello im having issues with install.bat. 
![image](https://github.com/noccu/umamusu-translate/assets/17797215/a7218dee-564f-49e6-9a50-9accb76be281)

**Comment by rukaiui** [comment]
i am running into the same issue i also reinstalled python, same version as before and also older versions like 3.9 and running into the same error. I will also attach a full log to this post as a comment so the whole executiuon is viewable:
[log.txt](https://github.com/noccu/umamusu-translate/files/13058651/log.txt)
i hope this helps with finding the issue

**Comment by noccu** [comment]
This is an issue with using python 3.12, as hinted at in the install guide. It is still new and many libraries have not updated their distributions yet, causing the install process to try to build them locally. This will fail on PCs not set up for development.

The solution is to use python 3.11 at most. When doing so, first the `.venv` folder in the patch should be deleted before running install.bat again. Python 3.12 should also be uninstalled, or if you need it for something else you can create the venv yourself by opening a cmd in the patch folder (you can simply write "cmd" in the windows explorer address bar and hit enter), paste `py -3.11 -m venv .venv` and enter (-3.11 can be replaced with the version you have installed), then running install.bat again.

In short:
1. delete the .venv folder from the patch
1. uninstall python 3.12 and install 3.11
1. run install.bat again

**Comment by wesh92** [comment]
From what I can tell this is not an issue any longer. Using Python 3.12 everything proceeded as expected!
